<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushBAddClient">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить клиента</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBAddFilial">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить филиал</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBAddAgr">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>290</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заключить договор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBDelClients">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>490</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить клиента</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBDelFilial">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>570</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить филиал</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBDelAgr">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>410</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Расторгунуть договор</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableAgr">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>1051</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>№ договора</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>№ паспорта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Код филиала</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата заключения</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Код страхов.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Наим. страхов.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Срок действия</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма(руб/месяц)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableClient">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>410</y>
      <width>661</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>№ паспорта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кол-во договоров</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableFilials">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>831</width>
      <height>171</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Код филиала</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Наим. филиала</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Адрес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>490</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>570</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Клиенты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Филиалы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Договора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер договора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Номер паспорта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>540</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Код филиала</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1162</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Страховая компания</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
